Services/DataX.Flow/DataX.Flow.InteractiveQuery/KernelService.cs

```diff
@@ -323,7 +335,18 @@ private void LoadFunctions(IKernel kernel, List<FunctionObject> functions)
                 if(fo.TypeDisplay=="UDF")
                 {
                     PropertiesUD properties = JsonConvert.DeserializeObject<PropertiesUD>(fo.Properties.ToString());
-                    string realPath = UDFPathResolver(properties.Path);
+                    string realPath = "";
+
```

Comment:
pattern comes back many times, refactor? and unit test?

Reply:
done